rails-commenting.rb

```diff
@@ -6,39 +6,39 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# ---1)
+# ---1)this is the controller file . its job is to hold all of the http verbs and their actions.
 class BlogPostsController < ApplicationController
   def index
-    # ---2)
+    # ---2)its creating a page that shows all of the instances for all blog post that are created .
```

Comment:
Great! This is referencing the instance variable that as you said, will show all the instances for blog posts.